### DISCUSSION TITLE
Channel behaviour analysis

### DISCUSSION THREAD

**Discussion by johnsonelahi** [rrirower/DP2SceneGraphWizard#87] [Q&A]
Hello, anybody can help me in this regard. Also there is no error in static analysis.
![image](https://github.com/rrirower/DP2SceneGraphWizard/assets/95589772/688b9b70-699a-4037-a1d4-7a842c5b123b)

**Comment by rrirower** [comment]
Please refer to discussion #38

**Reply by rrirower** [reply]
Have you resolved this?

**Reply by johnsonelahi** [reply]
The issue not resolved yet.

**Reply by rrirower** [reply]
Please check with the latest version 1.0.1.2.

**Reply by wajidbux** [reply]
I've checked with 1.0.1.2, I made two packages, one with ads and one without ads. The one with no ads passes the certification and gets published, but the one with ads passes four of the behaviour tests but fails the content play performance (3.6) with error message " content should start playing video within 8 seconds of initiation" but in debug console VOD start time is less than 4 seconds. I'm using CloudFront as a CDN. I know it sounds weird, but I've tested it over 50 times.

**Reply by rrirower** [reply]
What are you using for ads?

**Reply by wajidbux** [reply]
vast tags

**Reply by rrirower** [reply]
What service?  And, are you using preroll ads?

**Reply by wajidbux** [reply]
Service from Nexxen, no preroll, only midroll with intervals of 10 mins, keeping in mind the RAF criteria.

**Reply by rrirower** [reply]
Have you tried using the Roku test ad server for your criteria test?  If it passes, you can then switch back to your ad server.

**Reply by wajidbux** [reply]
No, I'm not aware of it. what is the Url?

**Reply by rrirower** [reply]
Try: https://devtools.web.roku.com/samples/sample.xml

**Reply by wajidbux** [reply]
I tried the Roku test ad server, still the same problem.

**Reply by rrirower** [reply]
Short of trying another CDN, I'm not sure how else to proceed.  At a minimum, you should contact Roku Partner Success and see if they can provide help.  I don't think there's a bug in the code, since the timing you report for debug seems reasonable.  And, the code is pretty straight forward: Select a content item and play it.  The code uses Roku modules to play the  content.

**Reply by rrirower** [reply]
Are you still dealing with this?

**Reply by wajidbux** [reply]
Yes, the problem persists, but it is unrelated to the code. I contacted Partner Success, they checked it at their end, and published the app  They did not tell me the reason why it was happening, but I assume it's a fault with Roku's 14.5 update, because I've noticed after the update, there are a lot of issues ranging from publishing to channelstore. For now, this is the way to publish an app while CBA is getting updated.

**Reply by rrirower** [reply]
Thanks for the update.
